arrow/arrow.py

```diff
@@ -1653,7 +1652,11 @@ def _get_datetime(
 
     @classmethod
     def _get_frames(cls, name: _T_FRAMES) -> Tuple[str, str, int]:
+        """Finds relevant timeframe and steps for use in range and span methods.
+
+        Returns a 3 element tuple in the form ("day", "days", 1.)
```

Comment:
Should we add the word example here so it is obvious this is an example output tuple and not the only one returned?